src/components/Tabs/Tab.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Tab = ({ clickTabItem, activeTab, label }) => {
+  let className = "tab-list-item";
```

Comment:
as we are using styled components. you don't have to use className for active status, but you can simply pass the active status as a prop to the tab like
```
const TabComponent = styled.li`
   ... all your styles
   ${({ active }) => active && css`
    ....all the active styles
   `}
`
```